source/documentation/deploying_services/index.md

```diff
@@ -4,26 +4,37 @@
 
 Many 12-factor applications rely on backing services such as a database, an email delivery service or a monitoring system. Routing services can be used to proxy and perform preprocessing on application requests such as caching, rate limiting or authentication.
 
-In Cloud Foundry, backing and routing services are referred to as 'services' and are available through the Cloud Foundry ``cf marketplace`` command.
+In Cloud Foundry, backing and routing services are referred to as 'services' and are available through the Cloud Foundry ``cf marketplace`` command. GOV.UK PaaS enables you to create a database service and bind it to your app. 
 
 Currently, the following database services are available in the marketplace:
 
  - PostgreSQL
  - MySQL
  - MongoDB (currently only available on private beta)
 
-
-Each service in the marketplace can have multiple plans available. For example, there are different PostgreSQL plans which vary by availability, storage capacity and encryption.
+Each service in the marketplace can have multiple plans available with different characteristics. For example, there are different PostgreSQL plans which vary by availability, storage capacity and encryption.
 
 Users can also define their own external services that are not available in the marketplace by using [User-Provided Service Instances](/#user-provided-service-instance).
 
+## PostgreSQL
+
+PostgreSQL is an object-relational database management system. It is open-source and designed to be extensible; currently the postgis and uuid-ossp extensions are enabled.
+
+## MySQL
+
+MySQL is an open source relational database management system that uses Structured Query Language (SQL) and is backed by Oracle.
+
+## MongoDB
```

Comment:
I agree on the parent-child headings issue and recognised this at time of drafting, but I also couldn't think at the time of a better way to organise this information given the LH Nav requirement. If we include H3s in the LH Nav, then it will be really long. @tomskerous would you be willing to have the database names as H3s?